locales/en.json

```diff
@@ -461,5 +463,25 @@
   "Long-term care costs": "Long-term care costs",
   "Public or non-profit long-term care home": "Public or non-profit long-term care home",
   "Did you pay property tax?": "Did you pay property tax?",
-  "Property tax paid": "Property tax paid"
-}
+  "Property tax paid": "Property tax paid",
+  "Register to vote": "Register to vote",
+  "Would you like to register to vote in the federal election?": "Would you like to register to vote in the federal election?",
+  "The Canada Elections Act gives rules about elections and voter information. Provincial and territorial governments, members of Parliament, registered political parties, and candidates at election time can have your voter information.": "The Canada Elections Act gives rules about elections and voter information. Provincial and territorial governments, members of Parliament, registered political parties, and candidates at election time can have your voter information.",
+  "If you complete this section, you will be registered to vote until you file your next tax return.": "If you complete this section, you will be registered to vote until you file your next tax return.",
+  "I allow CRA to give my name, address, date of birth and citizenship status to Elections Canada. Elections Canada uses the information to update voter records.": "I allow CRA to give my name, address, date of birth and citizenship status to Elections Canada. Elections Canada uses the information to update voter records.",
+  "Check the boxes if:": "Check the boxes if:",
+  "test": "test",
+  "test2": "test2",
```

Comment:
Not sure all these keys are getting used. 

I would say:
- delete all the new keys
- run through the pages again in dev mode

Then the "good" keys will be regenerated.